libs/core---samd/platform.h

```diff
@@ -135,4 +139,72 @@ typedef int PinName;
 #define PB30 62
 #define PB31 63
 
-#endif
\ No newline at end of file
+// 128 pin packages
+#ifdef SAMD51
+#define PC00 = 64
```

Comment:
There is no `=` in #define syntax, you'll need to remove them

Reply:
https://github.com/lalauria2000/pxt-common-packages/commit/f51b311ec37b114913bec683ee76ad55de341040